_opensearch/rest-api/analyze-apis/terminology.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Analysis API Terminology
+parent: Analyze APIs
+grand_parent: REST API reference
+nav_order: 1
+---
+
+# Terminology
+
+The following sections provide descriptions of important text analysis terms. 
+
+## Analyzers
+
+Analyzers instruct OpenSearch how to index and search text. Analyzers comprise three components: a tokenizer, zero or more token filters, and zero or more character filters. 
+
+OpenSearch provides *built-in* analyzers. For example, the `standard` built-in analyzer converts text to lower case and breaks text into tokens based on word boundaries such as carriage returns and white space. The `standard` analyzer is also called the *default* analyzer and is used when no analyzer is specified in the text analysis request.
+
+If needed, you can combine tokenizers, token filters, and character filters to create a *custom* analyzer.
+
+#### Tokenizers
+
+Tokenizers break unstuctured text into tokens and maintain metadata about tokens such as their start and ending positions in the text.
+
+#### Character filters
+
+Character filters examine textand perform translations such as changing, removing, and adding characters. 
+
+#### Token filters
+
+Token filters modify tokens, performing operations such converting a token's characters to upper case, and adding or removing tokens. 
```

Comment:
"...such as converting..."
Can probably ditch the comma before "...and adding..."